Cargo.toml

```diff
@@ -11,25 +11,15 @@ homepage = "https://cosmwasm.com"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cosmwasm-schema = "1.0.0"
-cosmwasm-std = "1.0.0"
-cw2 = "0.16"
+cosmwasm-schema = "1.1.9"
+cosmwasm-std = "1.1.9"
+cw2 = "1.0.0"
 schemars = "0.8.1"
 semver = "1"
 serde = { version = "1.0.0", default-features = false, features = ["derive"] }
 thiserror = "1.0.13"
 
-# We don't use the following dependencies directly. They're dependencies of our dependencies.
-# We specify them to tighten their version requirements so that builds with `-Zminimal-versions` work.
-# Once we bump `cosmwasm-*` deps to a version after `1.1.5`, we can remove these.
-k256 = { version = "0.11.1", features = ["ecdsa"] }
-anyhow = "1.0.41"
-
 [dev-dependencies]
-cw-storage-plus = "0.16"
-prost = "0.9"
-
-# We don't use the following dependencies directly. They're dependencies of our dependencies.
-# We specify them to tighten their version requirements so that builds with `-Zminimal-versions` work.
-# Once we bump `cosmwasm-*` deps to a version after `1.1.5`, we can remove these.
-serde_json = "1.0.40"
+cw-storage-plus = "1.0.1"
+prost = "0.11.0"
```

Comment:
```suggestion
prost = "0.11"
```

Reply:
The problem with those changes it it makes the patch release invisible. This is often the cause for setting the min version of a dependency too low. Of there is `.0` you see that you either want .0 or that you want something else.

Reply:
IMO it doesn't make them invisible - it literally means the same. I would go even further, some users who don't understand how Cargo dep versioning works might be confused.
I'm good either way. :+1: